sdk/core/azure-core/inc/azure/core/http/policy.hpp

```diff
@@ -217,6 +245,22 @@ namespace Azure { namespace Core { namespace Http {
     }
   };
 
+  /**
+   * @brief The options for the #TelemetryPolicy
+   *
+   */
+  struct TelemetryPolicyOptions
+  {
+    /**
+     * @brief The Application id is the last part of the user agent for telemetry.
```

Comment:
nit:
```suggestion
     * @brief The Application ID is the last part of the user agent for telemetry.
```

Reply:
We tend to use 'Id' on all our types, i.e.  'ApplicationId'.  I prefer Id.  Whichever we could choose we need to be consistent in documentation and code.